docs/docs/guides/core-types/datasets.md

```diff
@@ -69,55 +78,114 @@ This guide will show you how to:
-        {'id': '1', 'sentence': "She goed to the store.", 'correction': "She went to the store."},
-        {'id': '2', 'sentence': "They plays video games all day.", 'correction': "They play video games all day."}
-    ])
-    dataset = Dataset.from_pandas(df)
-    df2 = dataset.to_pandas()
+  ### Pandas DataFrame
 
-    assert df.equals(df2)
-    ```
+  To create a dataset from a Pandas `DataFrame` object, use the `from_pandas` method. 
 
-    </TabItem>
+  To convert the dataset back, use `to_pandas`.
 
-  </Tabs>
+  ```python
+  import pandas as pd
 
-  </TabItem>
-  <TabItem value="typescript" label="TypeScript">
+  df = pd.DataFrame([
+      {'id': '0', 'sentence': "He no likes ice cream.", 'correction': "He doesn't like ice cream."},
+      {'id': '1', 'sentence': "She goed to the store.", 'correction': "She went to the store."},
+      {'id': '2', 'sentence': "They plays video games all day.", 'correction': "They play video games all day."}
+  ])
+  dataset = Dataset.from_pandas(df)
+  df2 = dataset.to_pandas()
 
-```typescript
-This feature is not available in TypeScript yet.  Stay tuned!
-```
+  assert df.equals(df2)
+  ```
 
   </TabItem>
+  <TabItem value="typescript" label="TypeScript">
+   This feature is not currently available in TypeScript.  Stay tuned!
+  </TabItem>
 </Tabs>
+
+## Edit and delete a dataset in the UI
+
+:::tip
+To follow along with the example screenshots shown in this section, run the code shown in the [Dataset quickstart](#dataset-quickstart) and navigate to the **Datasets** tab in the Weave UI.
+:::
+
+You can edit and delete existing datasets from the **Datasets** tab in the UI. To create a dataset, [use one of the SDKs](#dataset-quickstart). 
+
+### Edit a dataset 
+
+1. Navigate to the Weave project containing the dataset you want to edit.
+2. From the sidebar, select **Datasets**. Your available datasets display.
+
+   ![Dataset UI](./imgs/datasetui.png)
+
+3. In the **Object** column, click the name and version of the dataset you want to edit. A pop-out modal showing dataset information like name, version, author, and dataset rows displays.
+
+   ![View dataset information](./imgs/datasetui-popout.png)
+
+4. In the upper right-hand corner of the modal, click the **Edit dataset** button (the pencil icon). An **+ Add row** button displays at the bottom of the modal.
+
+    ![Dataset UI- Add row icon](./imgs/datasetui-popout-edit.png)
+
+5. Click **+ Add row**. A green row displays at the top of your existing dataset rows, indicating that you can add a new row to the dataset. 
+
+    ![Dataset UI](./imgs/datasetui-popout-edit-green.png)
+
+6. To add data to the new row, click the column in the new row that you want to edit. You can't edit the default **id** column in a dataset row, as this is automatically assigned by Weave upon creation. A pop-out modal for editing displays, with **Text**, **Code** and **Diff** options for formatting.
```

Comment:
I find the last sentence on this line difficult to parse. Maybe `When an editable cell is clicked, a cell editor pop-out appears. The editor supports ... format options.`